docs/content/manual/release-specific/v1.3.0/test-storage-network.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Setup and test storage network
+---
+
+## Related issue
+https://github.com/longhorn/longhorn/issues/2285
+
+## Test storage network
+
+### Create AWS instances
+**Given** Create VPC.
+- VPC only
+- IPv4 CIDR 10.0.0.0/16
+
+*And* Create an internet gateway.
+- Attach to VPC
+
+*And* Add the internet gateway to the VPC `Main route table`, `Routes`.
+- Destination 0.0.0.0/0
+
+*And* Create 2 subnets in the VPC.
+- Subnet-1: 10.0.1.0/24
+- Subnet-2: 10.0.2.0/24
+
+*And* Launch 3 EC2 instances.
+- Use the created VPC
+- Use subnet-1 for network interface 1
+- Use subnet-2 for network interface 2
+- Disable `Auto-assign public IP`
+- Add security group inbound rule to allow `All traffic` from `Anywhere-IPv4`
+- Stop `Source/destination check`
```

Comment:
We are setting our own routes for the instances, and according to the [doc](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_ModifyNetworkInterfaceAttribute.html) we need to have it disabled. I think something to do with AWS expecting MAC and instance IP to match.

```
SourceDestCheck
Enable or disable source/destination checks, which ensure that the instance is either the source or the destination of any traffic that it receives. If the value is true, source/destination checks are enabled; otherwise, they are disabled. The default value is true. You must disable source/destination checks if the instance runs services such as network address translation, routing, or firewalls.
```